sass/_imported.scss

```diff
@@ -251,5 +432,11 @@ img.fullimage {
     .sponsors img, .sponsors li {
         font-size: 0.75em;
     }
-}
 
+    .about-paragraphs .eight:first-child {
+        padding-right:0;
+    }
+    .about-paragraphs .eight:nth-child(2) {
+        padding-left:0;
+    }
+}
```

Comment:
Sorry I hadn't been clearer about this, but could you move all about-page related scss into its own file?`_imported.scss` contains holdover from the original template, and as you can tell it's pretty unorganized.

Instead, create something like `_about.scss` in the `sass` directory, then add another import statement at the bottom of `mbx.scss`. The underscore in the filename tells Sass not to compile a standalone `about.css`, but rather use it as an import for other files. So importing a stylesheet into `mbx.scss` causes it to end up in the generated `mbx.css`.